spring-cloud-data-flow/spring-cloud-data-flow-stream-processing/data-flow-server/src/test/java/org/baeldung/SpringContextLiveTest.java

```diff
@@ -1,16 +0,0 @@
-package org.baeldung;
```

Comment:
We already had a [ContextIntegrationTest](https://github.com/eugenp/tutorials/blob/master/spring-cloud-data-flow/spring-cloud-data-flow-stream-processing/data-flow-server/src/test/java/org/baeldung/SpringContextIntegrationTest.java), and since this project doesn't need any external setup to start the context, I removed the LiveTest